doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+* E - ez390 emulator runtime which loads and executes 390 load module
+
+!!! Note "Use of parameters with parentheses in *nix shells"
+    Unix shells like sh and bash interpret parentheses if not quoted. For this reason, if you need to specify a z390 parameter that contains
+    parentheses, then that parameter should be quoted. For example, the SYSMAC parameter is quoted as it uses parentheses, whereas the ALIGN parameter is not.
+
+        "SYSMAC(foldername)" ALIGN 
+
+Option | Scope | Default | Description  
+-------|-------|---------|--------------
+@filename	| MALE	| Options in Z390.OPT file if found. |	Retrieve additional options from free form text file with default suffix OPT.  Options can be specified delimited by spaces on as many lines as requires.  All characters on a line following * are ignored as comments. The @file option can be nested.  The default path is the program path.  If a file named Z390.OPT exists in the z390 install directory, these options will be applied first as default options.
+ALIGN | A | YES	| Align DS/DC data fields based on type unless explicit length is specified.  If duplication factor is 0, then NOALIGN is ignored.
+ALLOW | M | NO | Allow extensions to HLASM syntax including: <br />1) No quotes required for SETC variables.<br />2) Duplication factor does not require (..).<br />3) Array declarations with variables in expression accepted<br />4) Substring starting beyond end returns null string<br />5) Quoted strings allowed in SETA and SETB expressions<br />6) Allow &var as null string if not found during substitution<br />7) Allow duplicate local and global variable declarations<br />8) Allow AREAD and PUNCH file records greater than 80 characters. <br />The default of NOALLOW insures HLASM compatibility.
+AMODE24 | LE | NO | Set 390 load module options to start in 24 bit address mode
+AMODE31 | LE | YES | Set 390 load module options to start in 31 bit address mode
+ASCII | AE | NO | Generate ASCII versus EBCDIC DC character constants, compare character strings in ASCII versus EBCDIC in macro processor, generate ASCII versus EBCDIC output for UNPK, ED, and EDMK.    Note ASCII mode is not mainframe compatible and requires careful review of program to insure no EBCDIC immediate compare constants such as X'40', 64, X'F0', 240 etc. are used and that there are no assumptions about EBCDIC versus ASCII collating sequences which have numbers and letters reversed, and EBCDIC letters are non-contiguous.  Typically HLASM compatible programs are run in default EBCDIC mode and DCB RECFM=FT?VT or some other translation option is used to convert between EBCDIC and ASCII where necessary.  Note there are a few instructions affected by ASCII mode: <br />1) ED/EDMK generate ASCII versus EBCDIC but the mask must always be in EBCDIC (masks are usually coded in X'...' format)<br />2)  UNPK generates X'3' versus X'F' in zone field (high order nibble)
```

Comment:
1. Remove extra spaces between 1st and 2nd sentence. "... EDMK.    Note ..." -> "... EDMK. Note ...".
2. "... RECFM=FT?VT ..." -> "... RECFM=FT|VT ...". Anthony: will the vertical bar mess up markdown?